ocaml/xapi/rfb_client_msgtype_parser.ml

```diff
@@ -0,0 +1,307 @@
+(*
+ * Copyright (C) Cloud Software Group
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU Lesser General Public License as published
+ * by the Free Software Foundation; version 2.1 only. with the special
+ * exception on linking described in file LICENSE.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Lesser General Public License for more details.
+ *)
+
+open Angstrom
+
+module D = Debug.Make (struct let name = "rfb_parser" end)
+
+open D
+
+(* RFB Message Types *)
+type rfb_message_type =
```

Comment:
This type name could be shorter. The module already has a long name, so `msg_type` or even `msg` would be enough to understand this.